crates/frontend/src/components/context_card.rs

```diff
@@ -37,37 +45,41 @@ pub fn context_card(
     ];
 
     view! {
-        <div class="rounded-lg shadow bg-base-100 p-6 shadow">
+        <div class="rounded-lg shadow bg-base-100 p-6 shadow flex flex-col gap-4">
```

Comment:
preferred order for classes:
"custom-non-tailwind classes"
"hidden"
"position z top/left/.."
"h w"
"m"
"p"
"flex props"
"text props"
"bg props"
"border props/rounded"
"shadow"
"transition/animation"